### PR TITLE
PT-755 Update loop mode naming

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: locales/fr/translation.json [hunk 1]
```diff
@@ -365,11 +365,11 @@
   "Looks like you've already sent an invitation to that email.": "Il semblerait que vous avez déjà envoyé une invitation sur cet email",
   "Looks like your e-mail address has not been verified.": "Il semblerait que votre adresse email n'a pas été vérifiée",
   "Loop mode": "",
-  "Loop mode off": "Loop mode désactivé",
+  "Loop mode off": "Loop mode arrêté",
   "low": "bas",
   "Low": "",
   "Lower target must be less than upper target.": "La limite basse doit être inférieure à la limite haute",
-  "M_Label": "D",
+  "M_Label": "S",
```
Comment (yvon-dblg) [inline locales/fr/translation.json]: From the Jira issue:

> as well as the letter "D" > "A"
```suggestion
  "M_Label": "A",
```

File: locales/de/translation.json [hunk 2]
```diff
@@ -365,11 +365,11 @@
   "Looks like you've already sent an invitation to that email.": "Sieht so aus, als hättest du bereits eine Einladung zu dieser E-Mail geschickt.",
   "Looks like your e-mail address has not been verified.": "Sieht so aus, als ob Ihre E-Mail-Adresse nicht verifiziert wurde.",
   "Loop mode": "Loop modus",
-  "Loop mode off": "Loop modus deaktiviert",
+  "Loop mode off": "Loop modus gestoppt",
   "low": "niedrig",
   "Low": "Niedrig",
   "Lower target must be less than upper target.": "Das untere Ziel muss kleiner als das obere Ziel sein.",
-  "M_Label": "O",
+  "M_Label": "S",
```
Comment (yvon-dblg) [inline locales/de/translation.json]: Why "S" and not "G" for example ?

File: locales/en/translation.json [hunk 3]
```diff
@@ -613,6 +613,7 @@
   "Uploaded on": "",
   "Upper target must be greater than lower target.": "",
   "Using Dexcom G5 Mobile on Android? See your data in Tidepool.": "Using Dexcom G6 Mobile on Android? See your data in Yourloops.",
+  "Using Tidepool": "",
```
Comment (yvon-dblg) [inline locales/en/translation.json]: Where does this comes from ?

File: locales/de/translation.json (hunk 2 above)
Comment (franck-fourel) [reply locales/de/translation.json]: Christian recommendation

File: locales/en/translation.json (hunk 3 above)
Comment (franck-fourel) [reply locales/en/translation.json]: it is displayed in the footer

**Pull request merged**